app/templates/_labtemplates/_js/modules/ui/test.js

```diff
@@ -0,0 +1,17 @@
+// define a new class
+function YourClass($element) {
```

Comment:
weten we zeker dat we dit willen doen? Met prototypes en alles?

Reply:
Nee. Suggesties?

Reply:
Niet echt, verder prima zo hoor 👍🏻